build/AndroidSdkLicenses/android-googletv-license

```diff
@@ -0,0 +1,2 @@
+
+601085b94cd77f0b54ff86406957099ebe79c4d6
```

Comment:
Will these ever expire and need to be re-approved periodically?

Reply:
Good question. Would be good to have a little readme about where to get these "magic" numbers, in case a new license gets added to the mix of these expire.